CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Hygieia Contribution Guidelines
+
+Thank you for your interest in contributing to Hygieia! It is because of developers like you that this project is able to succeed.
```

Comment:
`It is because of developers like you that this project is able to succeed.` --> `Developers like you are what allows this project to succeed`

Wording is slightly awkward, but not sure my suggestion is better.